src/m.camera.js

```diff
@@ -83,7 +83,7 @@ please.CameraNode = function () {
     this.__is_camera = true;
 
     please.make_animatable_tripple(this, "look_at", "xyz", [0, 0, 0]);
-    please.make_animatable_tripple(this, "up_vector", "xyz", [0, 0, 1]);
+    please.make_animatable_tripple(this, "up_vector", "xyz", please.gl.ctx === null ? [0, 1, 0] : [0, 0, 1]);
```

Comment:
In this case, we never want this property to be able to change, correct?  The conditional should instead be that please.make_animatable_tripple is only ever called when please.gl.ctx is defined, otherwise it should be set to [0, 0, 1] and have the property frozen or something.